src/components/Operations.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from 'react';
+import Snackbar from '@material-ui/core/Snackbar';
+import Alert from '@material-ui/lab/Alert';
+import Fade from '@material-ui/core/Fade';
+import CircularProgress from '@material-ui/core/CircularProgress';
+
+export default class Operations extends Component {
+    constructor(props) {
+		super();
+		this.state = {
+        }
+    }
+
+    oneLiner = (method) => {
+        //validate amount value
+        let amount = document.getElementById("amount");
```

Comment:
do not use the document, use the input event handler like what we showed in the kernel